src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle implements Figure {
+    private String color;
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        this.color = color;
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * base * height;
+    }
+
+    @Override
```

Comment:
The method `draw` should not start with an empty line. Please remove the empty line above the method declaration.